### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
PLease review my homepage

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,547 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```
Comment (junaidshaikh-js) [inline main.css:7]: `font-family` can be applied to the body tag, as it gets inherited, we don't need to add it to all the elements using universal selector.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+
+html {
+  font-size: 62.5%;
+  overflow-x: hidden;
+  scroll-padding-top: 6rem;
+  scroll-behavior: smooth;
+}
+
+:root {
+  /* --primary: #621a9a; */
```
Comment (junaidshaikh-js) [inline main.css:23]: code comment should be removed

**Review by vaibhav18matere — APPROVED** [review]
looks good. pretty descriptive class names and clean code throughout ! good to go

File: main.css [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+header .navbar a {
+  font-size: 1.7rem;
+  color: var(--bg-code);
+  margin: 0 1rem;
+}
+
+header .navbar a:hover {
+  color: var(--secondary-hover);
+}
+header .class-form {
```
Comment (junaidshaikh-js) [inline main.css:68]: Use consistent space between the code blocks

File: main.css [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+}
+.logo-header {
+  display: flex;
+  align-items: center;
+}
+.logo-design {
+  width: 4rem;
+}
+
+#menu-bar {
```
Comment (junaidshaikh-js) [inline main.css:115]: ids aren't recommended for styling purpose

File: main.css [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+  z-index: 0;
+}
+.home .slide-container .slide .image .travel {
+  width: 100%;
+  transform: rotate(0deg);
+  animation: shoeContent 0.3s linear;
+  border-radius: 5rem;
+  padding: 1rem 0;
+}
+.home .slide-container:nth-child(2) .slide .image .travel {
```
Comment (junaidshaikh-js) [inline main.css:165]: You can probably refractor this class to lower the specificity.

File: main.html [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+      </form>
+
+      <div class="font-icons">
+        <a href="" class="fas fa-heart"></a>
+        <a href="" class="fas fa-shopping-cart"></a>
+        <a href="" class="fas fa-user"></a>
+      </div>
+    </header>
+
+    <section class="home" id="home">
```
Comment (junaidshaikh-js) [inline main.html:67]: Can be nested inside the `main` landmark tag

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: main.html [hunk 7]
```diff
@@ -0,0 +1,362 @@
...
+              Get your Hiking Boots now at Best Price. For Limited Period Only.
+            </h3>
+            <a href="#" class="btn">Explore More</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="latest-brands" id="brands">
+      <h1 class="heading">Featured Brands</h1>
```
Comment (junaidshaikh-js) [inline main.html:221]: There should be only one `h1` element per HMTL page. Read about [heading here](https://www.contentkingapp.com/academy/headings/)

File: main.html [hunk 8]
```diff
@@ -0,0 +1,362 @@
...
+      <div class="brands-container">
+        <div class="brands-box">
+          <img src="./components/images/brands-1.jpg" alt="" />
```
Comment (junaidshaikh-js) [inline main.html:224]: `img` tag should have description `alt` text unless purely used for decoration purpose.

File: main.html [hunk 9]
```diff
@@ -0,0 +1,362 @@
...
+          veniam at itaque culpa hic corporis saepe dicta doloremque nihil.
+        </p>
+        <form action="">
+          <input type="email" placeholder="Enter your email" class="box" />
+          <input type="submit" value="Send" class="btn" />
+        </form>
+      </div>
+    </section>
+
+    <section class="footer">
```
Comment (junaidshaikh-js) [inline main.html:315]: There is native `footer` element that should be used for footer of the page

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

Some suggestions: 
- Use proper use of heading. 
- Don't use id to apply styles 
- Focus more on using HTML semantic elements.

**Pull request merged**